src/main/ansible/roles/testdata/tasks/main.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- name: Ensure /etc/ansible/facts.d exists
+  file:
+    path: /etc/ansible/facts.d
+    recurse: yes
+    state: directory
+
+- name: Copy facts-scripts to server
+  copy:
+    src: conditions.fact
+    dest: /etc/ansible/facts.d/
+    mode: "u+x"
+  register: result
+
+- name: Reload ansible_local
+  setup: filter=ansible_local
+  when: result.changed
+
+- name: Create link to test data
+  file:
+    src: /vagrant/src/test/resources/bags/basic-sequence-unpruned-with-refbags
```

Comment:
@rvanheest : why not to `/vagrant/src/test/resources/bags/`? Now we have access only to one set of test bags.

Reply:
You're right, that might be a better thing to do!